templates/vue/src/components/node-modal/CoordinateForm.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div>
+    <div id="modal-coordinate">
+      <b-form-group>
+        <b-form-checkbox v-model="isOnMap">
+          Show this node on map
+        </b-form-checkbox>
+      </b-form-group>
+      <b-form-group v-if="isOnMap">
+        <b-form-input
+          v-model="node.mapCoordinates.lat"
+          :number="true"
+          placeholder="enter latitude coordinate here"
```

Comment:
Don't forget to always capitalize. You could also shorten it a bit like "Enter latitude".